modules/transport/index.js

```diff
@@ -0,0 +1,27 @@
+import { API_ID } from '../API';
+
+import Transport from './Transport';
+
+/**
+ * Option for the default low level transport.
+ *
+ * @type {Object}
+ */
+const defaultTransportOptions = {};
+
+if (typeof API_ID === 'number') {
```

Comment:
Hmm. The problem here is that external_api.js uses Transport class as well but need to provide different options.